dnnsoftware/Dnn.Platform
# Removed the unused json2.js from the codebase

Fixes #2725 

## Summary
Removing the json2.js and associated license file from the codebase as it was only used for older browsers (IE7 and below). This file was primarily used in the 

## How to Test

1. Login either as a Super User or a regular user
2. Click on your username to view your profile
![image](https://user-images.githubusercontent.com/13650025/96306417-a3750680-0fcd-11eb-9569-66e6a1c88493.png)

3. You'll see a box with a placeholder "Tell the world something" 
![image](https://user-images.githubusercontent.com/13650025/96306601-fc449f00-0fcd-11eb-8969-382abdab6581.png)

4. Type a message and click Share to post it to the Journal 
![image](https://user-images.githubusercontent.com/13650025/96306529-d15a4b00-0fcd-11eb-9bb0-33da453a8e4c.png)

5. You should be able to post a message without any errors
![image](https://user-images.githubusercontent.com/13650025/96306575-e9ca6580-0fcd-11eb-94d3-3c1a1231163a.png)
